<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Persona Boy</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/Icons/persona.png</normalon>
   </iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>256</width>
    <height>256</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Image"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuSequence">
    <property name="title">
     <string>Sequence</string>
    </property>
    <addaction name="actionSet_Directory"/>
    <addaction name="separator"/>
    <addaction name="action_VideoFile"/>
    <addaction name="separator"/>
    <addaction name="action123_files_in_sequence"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionSelect_camera"/>
    <addaction name="actionOpen_camera"/>
    <addaction name="separator"/>
    <addaction name="actionNetwork_camera"/>
    <addaction name="actionKinect"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFullscreen"/>
    <addaction name="separator"/>
    <addaction name="m_actionShowControl"/>
   </widget>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>Processing</string>
    </property>
    <addaction name="m_actionProcDisable"/>
    <addaction name="separator"/>
    <addaction name="m_actionProcInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSequence"/>
   <addaction name="menuCamera"/>
   <addaction name="menuView"/>
   <addaction name="menuProcessing"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Image">
   <property name="text">
    <string>Open ...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionSet_Directory">
   <property name="text">
    <string>Image Directory ...</string>
   </property>
  </action>
  <action name="action_VideoFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Video File ...</string>
   </property>
  </action>
  <action name="m_actionShowControls">
   <property name="text">
    <string>show controls</string>
   </property>
  </action>
  <action name="action123_files_in_sequence">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>0 frames in sequence</string>
   </property>
  </action>
  <action name="actionSelect_camera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>select camera</string>
   </property>
  </action>
  <action name="actionOpen_camera">
   <property name="text">
    <string>usb camera ...</string>
   </property>
  </action>
  <action name="actionNetwork_camera">
   <property name="text">
    <string>network camera...</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="m_actionShowControl">
   <property name="text">
    <string>show controls</string>
   </property>
  </action>
  <action name="actionCCF_info">
   <property name="text">
    <string>show info</string>
   </property>
  </action>
  <action name="actionKinect">
   <property name="text">
    <string>kinect</string>
   </property>
  </action>
  <action name="m_actionProcDisable">
   <property name="text">
    <string>disable</string>
   </property>
  </action>
  <action name="m_actionProcInfo">
   <property name="text">
    <string>info</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="PersonaBoy.qrc"/>
 </resources>
 <connections/>
</ui>
